index.html

```diff
@@ -46,27 +45,27 @@ <h3 class="connect">LET’S CONNECT</h3>
           <ul class="linkSocials">
             <li>
               <a href="twitter.com">
-                  <img src="./Images/SocialMediaIcons/Twitter.png" alt="Twitter" />
+                  <img src="/Images/SocialMediaIcons/Twitter.png" alt="Twitter" />
```

Comment:
- [ ] [OPTIONAL] Your images look broken here in my browser. I don't know if they are visible on our side, you may check again locally but also on your deployed version. While I was trying to debug this, I realized that adding the dot before the `/` **will solve the bug** 😃. What I mean is that your src attribute looks like this, `src="./Images/SocialMediaIcons/Twitter.png" 💡 Please try to add it again if you find out that your images are broken. 🏃‍♂️